### PR TITLE
TODO items annotated with issue numbers

### PR DESCRIPTION
**Pull request by rajarshimaitra**

### CODE REVIEW

File: src/chain.rs [hunk 1]
```diff
@@ -27,7 +27,7 @@ pub type P2pMagicNumber = u32;
 /// Magic number prefixing Pubkey or Prvkey data according to BIP32 spec
 pub type Bip32MagicNumber = u32;
 
-// TODO: (new) Move constants to rust-bitcoin
+// TODO #186: (new) Move constants to rust-bitcoin
```
Comment (dr-orlovsky) [inline src/chain.rs:30]:
```suggestion
// TODO #186: Move constants to rust-bitcoin
```

File: src/chain.rs [hunk 2]
```diff
@@ -274,7 +274,7 @@ lazy_static! {
         p2p_port: 8333,
         rpc_port: 8332,
         ln_height: 504500,
-        // TODO: (new) update with first RGB release
+        // TODO #187: (new) update with first RGB release
```
Comment (dr-orlovsky) [inline src/chain.rs:277]:
```suggestion
        // TODO #187: update with first RGB release
```

File: src/chain.rs [hunk 3]
```diff
@@ -302,7 +302,7 @@ lazy_static! {
         p2p_port: 18333,
         rpc_port: 18332,
         ln_height: 1,
-        // TODO: (new) update with first RGB release
+        // TODO #187: (new) update with first RGB release
```
Comment (dr-orlovsky) [inline src/chain.rs:305]:
```suggestion
        // TODO #187: update with first RGB release
```

File: src/chain.rs [hunk 4]
```diff
@@ -376,7 +376,7 @@ lazy_static! {
     /// Liquid V1 chain parameters
     static ref CHAIN_PARAMS_LIQUIDV1: ChainParams = ChainParams {
         name: "liquidv1".to_string(),
-        // TODO: (new) check Liquid network magic number and change this if needed
+        // TODO #188: (new) check Liquid network magic number and change this if needed
```
Comment (dr-orlovsky) [inline src/chain.rs:379]:
```suggestion
        // TODO #188: check Liquid network magic number and change this if needed
```

File: src/tweak.rs [hunk 5]
```diff
@@ -35,7 +35,7 @@ pub enum ParseError {
     /// curve point ({0})
     WrongTweakFormat(String),
 
-    // TODO: (new) Once miniscript will do a better descriptor parse errors,
+    // TODO #190: (new) Once miniscript will do a better descriptor parse errors,
```
Comment (dr-orlovsky) [inline src/tweak.rs:38]:
```suggestion
    // TODO #190: Once miniscript will do a better descriptor parse errors,
```

File: strict_encoding/src/test_helpers.rs [hunk 6]
```diff
@@ -17,7 +17,7 @@ use std::io::{BufWriter, Write};
 
 use crate::{Error, StrictDecode, StrictEncode};
 
-// TODO: (new) Move into derive macro
+// TODO #196: (new) Move into derive macro
```
Comment (dr-orlovsky) [inline strict_encoding/src/test_helpers.rs:20]:
```suggestion
// TODO #196: Move into derive macro
```

**Review by dr-orlovsky — CHANGES_REQUESTED** [review]
Everythin is ok apart from `(new)` prefixes which I removed with suggestions

**Comment by rajarshimaitra** [comment]
Updated..

**Review by dr-orlovsky — APPROVED** [review]

**Pull request merged**